pkg/util/typeutil/schema.go

```diff
@@ -1601,6 +1602,46 @@ func CreateSparseFloatRowFromJSON(input map[string]interface{}) ([]byte, error)
 	return row, nil
 }
 
+func CreateSparseFloatRowFromJSON(input []byte) ([]byte, error) {
+	var indices []uint32
+	var values []float32
+
+	var vec sparseFloatVectorJSONRepresentation
+	decoder := json.NewDecoder(bytes.NewReader(input))
+	decoder.DisallowUnknownFields()
+	err := decoder.Decode(&vec)
```

Comment:
we should be able to remove `sparseFloatVectorJSONRepresentation` and avoid the duplicate parsing
```
var vec map[string]interface{}

err := decoder.Decode(&vec)
return CreateSparseFloatRowFromMap(vec)
```

Reply:
good